doc/mgr/nfs.rst

```diff
@@ -76,6 +76,16 @@ service.
 For more details, refer :ref:`orchestrator-cli-placement-spec` but keep
 in mind that specifying the placement via a YAML file is not supported.
 
+Since the deployment of NFS daemon(s) and the ingress service is asynchronous,
+check the statuses of the NFS and ingress services to make sure that the
+services are running. If your orchestrator is cephadm, then check using:
```

Comment:
Suggest reducing duplication, the "don't use  a word twice in a sentence" factor.  Something like:

Deployment of NFS daemons and the ingress service is asynchronous: the command may return before the services have completely started.  You may wish to check that these services do successfully start and stay running.  When using `cephadm` orchestration, these commands check service status: